blocks/melody/ff_module/ff_module-task-event-status/ff_module-task-event-status.less

```diff
@@ -1,19 +1,26 @@
+@base-height: 1.25rem;
+@visible: 1;
+@invisible: 0.01;
+
 .ff_module-task-event-status {
 
+    font-size: .8rem;
```

Comment:
Isn't it better to leverage less's power to calculate sizes like this? E.g., `font-size: unit(20/@base,rem);` ?